sympy/logic/tests/test_boolalg.py

```diff
@@ -758,107 +758,107 @@ def test_true_false():
     # Note: using 'is' comparison is important here. We want these to return
     # true and false, not True and False
```

Comment:
Why did you invert these tests? This comment explicitly states that these are using `is` on purpose. 

I don't understand why you changed `is` to `==` in this file. The tests using `is` are already fine, and the places that use `==` are the ones that shouldn't be changed (primarily the ones higher up in this test function).